tests/csm/rest/test_capacity_quota.py

```diff
@@ -1565,3 +1564,247 @@ def test_41970(self):
+        self.log.info("Set quota API response: %s", resp.json())
+        assert resp.status_code == HTTPStatus.OK, "Status code check failed"
+        self.log.info("Step 3: Perform GET API to get user level quota fields.")
+        res = self.csm_obj.get_user_quota(user_id)
+        assert res.status_code == HTTPStatus.OK, "Status code check failed"
+        self.log.info("Step 4: Verify the user level quota fields as per request.")
+        user_quota = res.json()
+        self.log.info("response: %s", user_quota)
+        assert_utils.assert_true(user_quota['enabled'], "Status check failed")
+        assert user_quota['max_size'] == max_size, "Max size field not matched"
+        assert user_quota['max_objects'] == max_objects, "Objects field not matched"
+        self.log.info("##### Test ended -  %s #####", test_case_name)
+
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-40601')
+    def test_40601(self):
+        """
+        Test that user can set and get the disabled User level quota/capacity fields for S3 user.
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Step 1: Perform POST API to create user.")
+        self.log.info("Creating IAM user payload.")
+        user_id, display_name = self.csm_obj.get_iam_user_payload()
+        payload = {"uid": user_id, "display_name": display_name}
+        self.log.info("payload :  %s", payload)
+        self.log.info("Creating IAM user.")
+        response = self.csm_obj.create_iam_user_rgw(payload)
+        assert response.status_code == HTTPStatus.CREATED, "Status code check failed"
+        self.log.info("Step 2: Perform PUT API to set user level quota fields.")
+        test_cfg = self.csm_conf["test_40601"]
+        max_size = test_cfg["max_size"]
+        max_objects = test_cfg["max_objects"]
+        enabled = test_cfg["enabled"]
+        quota_payload = {"enabled": enabled, "max_size": max_size,
+                         "max_objects": max_objects}
+        resp = self.csm_obj.set_user_quota(user_id, quota_payload)
+        assert resp.status_code == HTTPStatus.OK, "Status code check failed"
+        self.log.info("Step 3: Perform GET API to get user level quota fields.")
+        res = self.csm_obj.get_user_quota(user_id)
+        assert res.status_code == HTTPStatus.OK, "Status code check failed"
+        user_quota = res.json()
+        self.log.info("Step 4: Verify the user level quota fields as per request.")
+        assert_utils.assert_true(user_quota['enabled'], "Status check failed")
+        assert user_quota['max_size'] == max_size, "Max size field not matched"
+        assert user_quota['max_objects'] == max_objects, "Objects field not matched"
+        self.log.info("Step 5: Perform PUT API to set user level quota fields.")
```

Comment:
Can you make the logs aptly descriptive so that the difference between both set quota steps is clearer?

Reply:
ok

Reply:
Done.
ref: https://github.com/Seagate/cortx-test/pull/1621.